views/partials/top_nav.handlebars

```diff
@@ -41,6 +41,31 @@
         <a class="nav-link i18n" data-i18n="ftm-nav-volunteer" href="#">Volunteer</a>
       </li>
       -->
+      <li class="nav-item dropdown active">
+        <a class="nav-link dropdown-toggle i18n" href="#" id="share-dropdown" role="button"
+           data-toggle="dropdown" aria-haspopup="true" aria-expanded="false"
+           data-i18n="ftm-nav-share">
+          Share
+        </a>
+        <div class="dropdown-menu dropdown-menu-right" aria-labelledby="share-dropdown"
+             id="countries-dropdown-selector">
+          <a class="fb-xfbml-parse-ignore dropdown-item" onclick="window.open('https://www.facebook.com/sharer/sharer.php?u=http%3A%2F%2Fwww.findthemasks.com%2F&amp;src=sdkpreparse','popup', 'width=600,height=800');return false;" href="https://www.facebook.com/sharer/sharer.php?u=http%3A%2F%2Fwww.findthemasks.com%2F&amp;src=sdkpreparse">
+            <div class="icon icon-facebook"></div> Facebook
```

Comment:
The background of the facebook icon disappears, since we picked the same shade of blue for dropdowns + secondary header. We should probably pick a different color.

Reply:
updated to a lighter blue version of the fb icon